Foundation/URLSession/http/HTTPURLProtocol.swift

```diff
@@ -93,7 +93,7 @@ internal class _HTTPURLProtocol: _NativeProtocol {
             case (_, nil):
                 set(requestBodyLength: .unknown)
             }
-        } catch let e {
+        } catch {
```

Comment:
`e` is used inside the catch block for  the `errorCode(fileSystemError: e)` call so this should be left in